src/css/iceberg-explorer.css

```
@@ -0,0 +1,183 @@
+  --ice-accent: var(--product-color, #e2401b);
+  --ice-muted: var(--color-gray-70, #6b7280);
+  --ice-border: var(--panel-border-color, #d0d3d9);
+  --ice-panel: var(--panel-background, #f7f8fa);
+  --ice-pre: var(--pre-background, #f0f1f4);
+  --ice-code: var(--code-font-color, var(--body-font-color));
+  --ice-green: #1a7f5a;
+  --ice-blue: #2563c9;
+  --ice-yellow: #9a6b00;
+  --ice-purple: #7c3aed;
+  --ice-cyan: #0e7490;
+  font-size: var(--secondary-font-size, 0.9rem);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)